### DISCUSSION TITLE
How to change the background color for full screen

### DISCUSSION THREAD

**Discussion by chenxiaowei107** [Budibase/budibase#12771] [Help] [answered]
In a new page, I added a container, and in the custom CSS, I added the css code to modify the background。 and I set the container's width for 100%, but why is the final background color not fully screened.

![1](https://github.com/Budibase/budibase/assets/39343523/50a28b83-9bee-4b2c-aa67-971add050227)
![2](https://github.com/Budibase/budibase/assets/39343523/1223d1de-1902-411e-ad5a-16736d761af2)
![3](https://github.com/Budibase/budibase/assets/39343523/f48347ac-f964-4e1b-bc4c-813050ea590c)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @chenxiaowei107,

Can you go click on the screen inside your component tree and then select max on the width setting on the far right of the screen. I've attached a screenshot below to illustrate this.

<img width="1364" alt="Screenshot 2024-01-15 at 07 29 25" src="https://github.com/Budibase/budibase/assets/126772285/ae46876a-141c-4ca6-8def-0c26fd6a08f7">

I hope this helps.

**Comment by chenxiaowei107** [comment]
Thanks,It was solved。
